### PR TITLE
Adding method in API to get all entities relationships

### PR DESCRIPTION
**Pull request by Mun1z**
**Depends on #117 .**

Adding method in ML-Git API, which aims to list all relationships of all project entities. This method uses the Github API.

```
  def get_project_entities_relationships(self, config_repo_name, export_type=FileType.JSON.value, export_path=None):
      """Get a list of relationships for all project entities.

      Args:
          config_repo_name (str): The repository name where the metadata is located in GitHub.
          export_type (str): Choose the format of the return [default: json].
          export_path (str): Set the path to export metrics to a file.

      Returns:
            list of EntityVersionRelationships.
      """
```

**Usage example:** 
```python
from ml_git import api
github_token = ''
api_url = 'https://api.github.com'
manager = api.init_entity_manager(github_token, api_url)

relationships = manager.get_project_entities_relationships(config_repo_name='user/metadata_repository')
```

The variable `linked_entities ` should look something like:
```
>>> manager.get_project_entities_relationships('lucasncabral/mlgit-lucas')
{'model-ex': [{
  "version": 1,
  "tag": "test__model-ex__1",
  "relationships": [
    {
      "tag": "test__dataset-ex__1",
      "name": "dataset-ex",
      "version": "1",
      "entity_type": "dataset"
    },
    {
      "tag": "test__labels-ex__1",
      "name": "labels-ex",
      "version": "1",
      "entity_type": "labels"
    }
  ]
}],
'labels-ex': [{
  "version": 1,
  "tag": "test__labels-ex__1",
  "relationships": [
    {
      "tag": "test__dataset-ex__1",
      "name": "dataset-ex",
      "version": "1",
      "entity_type": "dataset"
    }
  ]
}]}
 
```

### CODE REVIEW

File: docs/api/quick_start.md [hunk 1]
```diff
@@ -316,4 +316,15 @@ api_url = 'https://api.github.com'
...
+
+
+#### List relationships from all project entities
+```python
+from ml_git import api
+github_token = ''
+api_url = 'https://api.github.com'
+manager = api.init_entity_manager(github_token, api_url)
+
+relationships = manager.get_project_entities_relationships(config_repo_name='user/metadata_repository')
```
[the remark this answers is not in the document]
Comment (Mun1z) [reply docs/api/quick_start.md:329]: The config_repo_name and metadata_repo_name are the config and metadata names in the git repository, for example:

metadata_repo_name='user/metadata_repository'
config_repo_name='user/config_repository'

The repo_name was renamed, it was actually config_repo_name too.

**Review by gilMars — APPROVED** [review]

File: docs/api/README.md [hunk 2]
```diff
@@ -306,6 +306,17 @@ class EntityManager:
...
             export_path (str): Set the path to export metrics to a file.
 
+        Returns:
+            list of EntityVersionRelationships.
+        """
+    def get_project_entities_relationships(self, config_repo_name, export_type=FileType.JSON.value, export_path=None):
+        """Get a list of relationships for all project entities.
+
+        Args:
+            config_repo_name (str): The repository name where the metadata is located in GitHub.
```
Comment (lucasncabral) [inline docs/api/README.md:316]:
```suggestion
            config_repo_name (str): The repository name where the config file is located in GitHub.
```

File: docs/api/quick_start.md (hunk 1 above)
Comment (lucasncabral) [inline docs/api/quick_start.md:329]:
```suggestion
relationships = manager.get_project_entities_relationships(config_repo_name='user/config_repository')
```

**Review by lucasncabral — APPROVED** [review]

**Review by andersonpablosilva — APPROVED** [review]

File: docs/api/README.md [hunk 3]
```diff
@@ -306,6 +306,17 @@ class EntityManager:
...
 
+        Returns:
+            list of EntityVersionRelationships.
+        """
+    def get_project_entities_relationships(self, config_repo_name, export_type=FileType.JSON.value, export_path=None):
+        """Get a list of relationships for all project entities.
+
+        Args:
+            config_repo_name (str): The repository name where the config file is located in GitHub.
+            export_type (str): Choose the format of the return [default: json].
```
Comment (gpretto) [inline docs/api/README.md:317]:
```suggestion
            export_type (str): Set the format of the return [default: json].
```
Should we add what formats it is supported?

Comment (Mun1z) [reply docs/api/README.md]: Done on commit: https://github.com/HPInc/ml-git/pull/118/commits/fe70ce2c6baa2f91ac505d44727624d65dc17a48

**Review by gpretto — APPROVED** [review]

**Pull request merged**